shell/assets/translations/en-us.yaml

```diff
@@ -3449,6 +3452,21 @@ persistentVolume:
     controllerPublishSecretNamespace:
       label: Controller Publish Secret Namespace
       placeholder: e.g. default
+    drivers:
+      disk-csi-azure-com: Azure Disk (CSI)
+      file-csi-azure-com: Azure File (CSI)
+      driver-longhorn-io: Longhorn
```

Comment:
I think there should be just a couple more changes to the labels:

- the `driver.longhorn.io` should say that it's a CSI driver (the screenshot showed it's an in-tree plugin but it's not)
- the Harvester one should say it's a CSI as well
- Amazon EBS Disk in-tree plugin should say it's deprecated

Reply:
Done
![Screen Shot 2022-12-06 at 8 43 40 AM](https://user-images.githubusercontent.com/42977925/205957425-4d0d0edf-ca88-4edf-b2fd-62c1e192fbd8.png)